packit_service/models.py

```diff
@@ -2102,6 +2104,8 @@ class CoprBuildTargetModel(GroupAndTargetModelConnector, Base):
 
     scan = relationship("OSHScanModel", back_populates="copr_build_target")
 
+    identifier = Column(String, default="")
```

Comment:
Ok is not a big issue not having the default here, I can remove it. In both cases this could be wrong. I will not create the logic for populate this field correctly because I think it is not worth the time we will spend on it.